libnopegl/src/nopegl.h.in

```diff
@@ -104,7 +104,7 @@ struct ngl_node;
  */
 struct ngl_scene {
     void *internal;         /* reserved internal pointer */
-    struct ngl_node *root;  /* root node of the scene graph */
+    struct ngl_node *root;  /* root node of the scene graph, must be set with the ngl_scene_init_from_* functions */
```

Comment:
I would something like `set by ngli_scene_init_from_* functions (must not be set by the user)`. While at it, I would mention that the fields below are user writable.

Reply:
Branch reworked to use `ngl_scene_params`, so these commits were not relevant anymore and thus are removed in this last iteration.

The PR is now on top of https://github.com/NopeForge/nope.gl/pull/199